### PR TITLE
[P53] Move Player to Marker on Move

### PR DESCRIPTION
**Pull request by EpicPants90**
Player now moves to marker in the building, should the destination tile have a building.

### CODE REVIEW

**Comment by EpicPants90** [comment]
https://trello.com/c/gd38M1Ra/64-p53-move-player-to-marker-on-move

**Comment by alasdairhurst** [comment]
@EpicPants90 Name your PR using the right convention please

**Comment by EpicPants90** [comment]
Fixed but where the camera can move but player does not.

File: Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/CommanderUI.cs [hunk 1]
```diff
@@ -16,20 +16,21 @@ public class CommanderUI : MonoBehaviour
...
+    public Player _Player;
     public float _LiftedHeight;
     public float _LiftTime;
     public float _MoveTime;
 
     Collider _collider;
-    Collider _prevHovered;
     LerpPosition _lerpPosition;
     Vector3 _toGoTo;
-	HashSet<TileData> _reachableTiles;
```
Comment (alasdairhurst) [inline Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/CommanderUI.cs]: what is this edit for?

**Comment by alasdairhurst** [comment]
found a bug where the game is not compete :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer: :dancer:

**Pull request merged**